client/src/components/Sidebar/Sidebar.tsx

```diff
@@ -193,6 +193,9 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = ({
         }
         type="button"
         ref={mobileNavToggleRef}
+        data-controller="w-kbd"
+        data-w-kbd-key-value="mod+["
+        data-w-kbd-scope-value="global"
```

Comment:
Based on main comment, let's replace this with the 'target' attribute.

```suggestion
        data-w-kbd-target={isMobile ? 'element' : undefined}
```

Also, we should set the `aria-keyshortcuts` further up for completeness & accessibility.


```ts
  return (
    <>
      <button
        onClick={onClickOpenCloseToggle}
        aria-label={gettext('Toggle sidebar')}
        aria-expanded={visibleOnMobile ? 'true' : 'false'}
        aria-keyshortcuts="["
// ...
```